src/Microsoft.AspNetCore.Server.Kestrel/Microsoft.AspNetCore.Server.Kestrel.csproj

```diff
@@ -26,4 +26,21 @@
     <PackageReference Include="System.Threading.ThreadPool" Version="4.4.0-*" />
   </ItemGroup>
 
+  <ItemGroup>
+    <CodeGenInput Include="..\..\tools\Microsoft.AspNetCore.Server.Kestrel.GeneratedCode\FrameFeatureCollection.cs" />
+    <CodeGenInput Include="..\..\tools\Microsoft.AspNetCore.Server.Kestrel.GeneratedCode\KnownHeaders.cs" />
+    <CodeGenOutput Include="Internal\Http\FrameHeaders.Generated.cs" />
+    <CodeGenOutput Include="Internal\Http\Frame.Generated.cs" />
+  </ItemGroup>
+
+  <Target Name="GenerateCode"
+          Inputs="@(CodeGenInput)"
+          Outputs="@(CodeGenOutput)"
+          BeforeTargets="CoreCompile">
+    <Exec Command="dotnet restore"
```

Comment:
It is if I run a full build. This would support building just `Microsoft.AspNetCore.Server.Kestrel.csproj`. We might not need that though - I did it so I could quickly iterate from empty repo to getting this working :smile: